src/sass/footer-system/_footer-resources.scss

```diff
@@ -1,74 +1,67 @@
 // Copyright (C) 2018 The Trustees of Indiana University
 // SPDX-License-Identifier: BSD-3-Clause
 
-@use "../core" as *;
-
 .rvt-footer-resources {
-	padding-bottom: var(--rvt-spacing-sm);
-
-	&__heading {
-		font-size: var(--rvt-ts-18);
-		font-weight: var(--rvt-font-weight-medium);
-		margin-top: var(--rvt-spacing-md);
-	}
+	padding-block: var(--rvt-spacing-md);
 
-	&__text-block {
-		font-size: var(--rvt-ts-14);
-		margin-top: var(--rvt-spacing-sm);
+	.rvt-footer-social + & {
+		padding-block-start: 0;
 	}
 
-	&__list {
-		list-style: none;
-		margin-top: var(--rvt-spacing-xs);
-		padding-left: 0;
-	}
+	.rvt-footer-resources__content {
+		display: flex;
+		flex-wrap: wrap;
+		gap: var(--rvt-spacing-md);
 
-	&__list-item {
-		margin-top: var(--rvt-spacing-xxs);
-	}
+		> * {
+			flex-grow: 1;
+		}
 
-	&__list-item a {
-		color: var(--rvt-color-theme-text-strong);
-		font-size: var(--rvt-ts-14);
-		text-decoration: none;
+		a {
+			color: var(--rvt-color-theme-text);
+			text-decoration: none;
 
-		&:hover {
-			text-decoration: underline;
+			&:hover {
+				text-decoration: underline;
+			}
 		}
 	}
 
-	/**
-   * This class can be used on any button or button links needed for CTAs, etc.
-   */
+	.rvt-footer-resources__address {
+		margin-block-start: var(--rvt-size-2);
 
-	&__button {
-		background-color: var(--rvt-color-theme-background);
-		border-color: transparent;
-		color: var(--rvt-color-theme-accent);
-		justify-content: center;
-		margin-bottom: var(--rvt-spacing-sm);
-		width: 100%;
+		h3 {
+			color: var(--rvt-color-theme-text-accent);
+			font: var(--rvt-font-20-black);
+			text-transform: uppercase;
+		}
+
+		p {
+			margin-block: var(--rvt-spacing-xs) 0;
+		}
 	}
-}
 
-@include mq($breakpoint-md) {
-	.rvt-footer-resources {
-		padding-bottom: var(--rvt-spacing-lg);
-		padding-top: var(--rvt-spacing-lg);
+	.rvt-footer-resources__links {
+		display: flex;
+		flex-wrap: wrap;
+		gap: var(--rvt-spacing-md);
 
-		&__heading {
-			margin-top: 0;
+		> * {
```

Comment:
These columns of links should be equal width like a normal grid. Right now the width is being determined by the flex algorithm based on the size of the content inside leading to uneven column widths.
<img width="1304" height="273" alt="image" src="https://github.com/user-attachments/assets/e99c657a-231d-47c1-bef8-522228908617" />

Usually adding something like `> * { flex-basis: 0; }` along with the `flex-grow: 1` you've already set will force all flex children to be equal width, but I'll leave the solution to you.